kpet/data.py

```diff
@@ -327,6 +376,64 @@ def is_dir_valid(dir_path):
         """
         return os.path.isfile(dir_path + "/index.yaml")
 
+    def validate(self):
+        """
+        Check if any of the regexes are invalid.
+
+        Returns:
+            Raises a schema.Invalid exception when finding an invalid regex
+        """
+        host_types = list()
+        trees = list()
+        arches = list()
+
+        for host_name in self.host_types or []:
+            host_types.append(host_name)
+        trees = list(self.trees.keys())
+        arches = self.arches
+
+        error = ("Configuration error: One (or more) of {match} - {what} " +
```

Comment:
Hmm, I wouldn't say kpet-db is configuration, it's more of input data, or a database. So, please avoid categorizing this error message. Just drop `Configuration error: `.

Reply:
Dropped, it says "kpet.schema.Invalid: " before the error string anyways so it wasn't just wrong it was redundant